src/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by engine on 22/02/2017.
+ */
+require('./../style/main.scss');
+
+import 'whatwg-fetch';
+import $ from 'jquery';
+import Movies from './Movies';
+import Utils from './Utils';
+
+let sendInput = $('#sendInput');
+sendInput.on('click', () => {
+  let year = Utils.parseInput();
+
+  if(year) {
+    listTopMovies(Utils.fullUrl(year))
+  }
+});
+
+let inputYear = $('#inputYear');
+inputYear.keyup(function (event) {
+  if (event.keyCode == 13) {
```

Comment:
`===` ⚠️ 
No magic numbers allowed. ❗️ 
Externalise it to constant: `const ENTER = 13;`